uvicorn/protocols/websockets/websockets_impl.py

```diff
@@ -103,14 +103,14 @@ def __init__(
             max_size=self.config.ws_max_size,
             ping_interval=self.config.ws_ping_interval,
             ping_timeout=self.config.ws_ping_timeout,
-            server_header=None,
             extensions=extensions,
             logger=logging.getLogger("uvicorn.error"),
-            extra_headers=[
+        )
+        self.server_header = None
+        self.extra_headers = [
```

Comment:
Test `test_multiple_server_header` would fail in versions <10.4.

Reply:
![image](https://user-images.githubusercontent.com/10519440/199091336-2638e141-eafa-4aa8-848e-8954d51e33dd.png)